packages/documentation/src/components/Header.jsx

```diff
@@ -1,7 +1,140 @@
+/* eslint-disable react/jsx-key */
 import React from 'react';
-import { Link } from 'gatsby';
+import { graphql, Link, StaticQuery } from 'gatsby';
 import sidebarData from '../sidebar';
 import MobileNav from './MobileNav';
+import { Index } from 'elasticlunr';
+
+const maxDisplayedSearchResults = 5;
+
+function StringWithBoldQuery(props) {
+  const { string, queries } = props;
+  const lowerCasedQueries = queries.map(query => query.toLowerCase());
+  const lowerCasedString = string.toLowerCase();
+
+  const matchedQuery = lowerCasedQueries.find(query =>
+    lowerCasedString.includes(query),
+  );
+  if (!matchedQuery) {
+    return <>{string}</>;
+  }
+
+  const queryStartIndex = lowerCasedString.indexOf(matchedQuery);
+
+  return (
+    <>
+      {string.slice(0, queryStartIndex)}
+      <strong>
+        {string.slice(queryStartIndex, queryStartIndex + matchedQuery.length)}
```

Comment:
multiple words are treated as multiple queries and the component bolds the first matching query. 

![Screen Shot 2019-04-30 at 18 11 06](https://user-images.githubusercontent.com/4998130/57000969-661b7f80-6b73-11e9-99c5-05881b5ef7c3.png)
![Screen Shot 2019-04-30 at 18 11 14](https://user-images.githubusercontent.com/4998130/57000970-661b7f80-6b73-11e9-8c0b-7ff650c3f49f.png)